orb-jobs-agent/src/handlers/wifi_ip.rs

```diff
@@ -0,0 +1,60 @@
+use crate::job_system::ctx::{Ctx, JobExecutionUpdateExt};
+use color_eyre::{eyre::Context, Result};
+use orb_relay_messages::jobs::v1::JobExecutionUpdate;
+
+/// command format: `wifi_ip`
+#[tracing::instrument]
+pub async fn handler(ctx: Ctx) -> Result<JobExecutionUpdate> {
+    // Try to get the IP address of the currently connected WiFi interface
+    // First, try to get the default route interface
+    let route_output = ctx
+        .deps()
+        .shell
+        .exec(&["ip", "route", "get", "8.8.8.8"])
+        .await
+        .wrap_err("failed to get default route")?
+        .wait_with_output()
+        .await
+        .wrap_err("failed to get output for route command")?;
+
+    let route_info = String::from_utf8_lossy(&route_output.stdout);
+    
+    // Extract the interface name from the route output
+    let interface = route_info
+        .split_whitespace()
+        .skip_while(|&word| word != "dev")
+        .nth(1)
+        .unwrap_or("wlan0"); // fallback to common WiFi interface name
```

Comment:
Side note, we do need this command in jobs agent for sure? I think backend-status already provides this

Reply:
I am not certain, but it's proving useful for development with stage orbs